accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableinstaller/impl/AuthInstallerUserManagerPrefetchingImpl.java

```diff
@@ -140,6 +140,35 @@ private void refreshAuthorizableCacheIfNeeded(final String id, final Authorizabl
         }
     }
 
+    @Override
+    public void removeAuthorizable(final Authorizable authorizable) throws RepositoryException {
+        if (authorizable == null) {
```

Comment:
Is this ever called with `null`? I would rather go for throwing an exception in that case to not hide a programming error.

Reply:
done!